infrateam_first_responder.md

```diff
@@ -15,9 +15,50 @@ It can be helpful to scan the list of PRs generated in Slack.  If you notice som
 
 #### Deploy 'em
 
-Use the `sdr-deploy` CLI, from your laptop, to deploy all infrastructure projects via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy.
+Use the `sdr-deploy` CLI, from the sdr-infra server, to deploy all infrastructure projects via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy.
 
-Note that you will need to be sure you can ssh into each of the VMs from your laptop. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to use the `check_ssh` command to do this.)
+Note that you will need to be sure you can ssh into each of the VMs from sdr-infra. (See the [sdr-deploy README](https://github.com/sul-dlss/sdr-deploy/blob/main/README.md) for more about how to use the `check_ssh` command to do this.)
+
+##### 1. Setup sdr-infra for deploying
+
+###### Login with Multi-factor Authentication (MFA)
+
+From your laptop:
```

Comment:
I'd suggest distinguishing the one-time vs the every-time steps. Also, might want to put the one-time steps into sdr-deploy README instead of here.

Reply:
If the instructions are specific to sdr-deploy, putting them in the sdr-deploy README makes sense to me. I do think we need some doc changes here too to cover our other one-off deploys.